libs/chrome-ext/src/lib/utils.js

```diff
@@ -0,0 +1,26 @@
+
+
+const isYoutubeUrl = (url) => {
+  return /^(?:https?:)?(?:\/\/)?(?:youtu\.be\/|(?:www\.|m\.)?youtube\.com\/)/.test(url);
```

Comment:
Here the method is creating the regexp everytime it is called. It's better to create the regexp as a const outside the method so the method always reuse the same instance of the regexp (you can apply this comment to all the methods using a regexp)

Reply:
ups I didn't pay attention to it, now it's fixed.